RoddyCore/test/src/de/dkfz/roddy/knowledge/files/BaseFileTest.groovy

```diff
@@ -91,59 +91,70 @@ class BaseFileTest {
     }
 
     @Test
-    public void testCreationOfBaseFileWithSourceHelper() {
-        BaseFile.ConstructionHelperForSourceFiles helperObject = new BaseFile.ConstructionHelperForSourceFiles(mockedTestFilePath, mockedContext, null, null);
-        BaseFile instance = syntheticTestFileClass.newInstance(helperObject);
-        assert instance;
-        assert instance.executionContext == mockedContext;
-        assert instance.path == mockedTestFilePath;
+    void testCreationOfBaseFileWithSourceHelper() {
+        BaseFile.ConstructionHelperForSourceFiles helperObject = new BaseFile.ConstructionHelperForSourceFiles(mockedTestFilePath, mockedContext, null, null)
+        BaseFile instance = syntheticTestFileClass.newInstance(helperObject)
+        assert instance
+        assert instance.executionContext == mockedContext
+        assert instance.path == mockedTestFilePath
     }
 
     @Test
-    public void testConstructForCreationOfBaseFileWithSourceHelper() {
-        def obj = BaseFile.constructSourceFile(syntheticTestFileClass, mockedTestFilePath, mockedContext, null, null);
-        assert obj instanceof BaseFile;
+    void testConstructForCreationOfBaseFileWithSourceHelper() {
+        def obj = BaseFile.constructSourceFile(syntheticTestFileClass, mockedTestFilePath, mockedContext, null, null)
+        assert obj instanceof BaseFile
     }
 
+    @Ignore("What is the difference between generic creation and manual creation? What does this test add that is not already tested elsewhere?")
```

Comment:
The difference is:
GenericCreation is meant to be primarily used for files created during GenericMethod calls
ManualCreation is the helper used by workflow developers